checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/CheckstyleProfileExporter.java

```diff
@@ -121,14 +124,17 @@ private void appendTreeWalker(Writer writer,
             }
         }
         // append Treewalker filters
-        final String filtersXml = settings.getString(CheckstyleConstants.TREEWALKER_FILTERS_KEY);
+        final String filtersXml = settings
+                .get(CheckstyleConstants.TREEWALKER_FILTERS_KEY)
+                .orElse(null);
         writer.append(filtersXml);
 
         writer.append(CLOSE_MODULE);
     }
 
     private boolean isSuppressWarningsEnabled() {
-        final String filtersXml = settings.getString(CheckstyleConstants.CHECKER_FILTERS_KEY);
+        final String filtersXml = settings.get(CheckstyleConstants.CHECKER_FILTERS_KEY)
+                .orElse(null);
```

Comment:
in other parts of code you fight with `null` usage, but keep it here.
Please keep usage of Option as it comes from library, especially when context if this variable is very small.

Reply:
I have kept it in order to require the minimal change of this method. As you have said, code cleanup regarding modern Java syntax/features should be done in a separate PR, I'd see such a change as a separate task.